src/imgui/ImGuiSettings.cc

```diff
@@ -398,7 +398,16 @@ void ImGuiSettings::showMenu(MSXMotherBoard* motherBoard)
 		im::Menu("Misc", [&]{
 			ImGui::MenuItem("Configure OSD icons...", nullptr, &manager.osdIcons->showConfigureIcons);
 			ImGui::MenuItem("Fade out menu bar", nullptr, &manager.menuFade);
-			ImGui::MenuItem("Show status bar", nullptr, &manager.statusBarVisible);
+			im::Menu("Status bar...", [&]{
```

Comment:
Detail. For items that open a sub menu we usually don't append "..." (we only do that for items that open a new window). Or at least I believe that's the common convention.

Reply:
OK, will fix that.